geopandas/tests/test_plotting.py

```diff
@@ -116,6 +116,17 @@ def test_style_kwargs(self):
         ax = self.df.plot(column='values', markersize=10)
         assert ax.collections[0].get_sizes() == [10]
 
+        ax = self.df.plot(markersize='values')
+        assert (ax.collections[0].get_sizes() == self.df['values']).all()
+
+        ax = self.df.plot(column='values', markersize='values')
+        assert (ax.collections[0].get_sizes() == self.df['values']).all()
+
+    def test_style_kwargs(self):
+
+        ax = self.points.plot(edgecolors='k')
+        assert (ax.collections[0].get_edgecolor() == [0, 0, 0, 1]).all()
```

Comment:
This seems to serve the purpose of verifying that kwargs are passed through, though maybe it was really intending to verify that `values`/`markersize` are correctly put within `kwargs` passed to `ax.scatter`, in which case maybe this isn't testing what it is supposed to test.

Reply:
I think it was testing both somewhat -> check that kwargs are passed through, and specifically test that markersize was working as this is a very useful one. 
So but I think those changes are fine!